internal/cmd/local_server.go

```diff
@@ -1,118 +0,0 @@
-package cmd
```

Comment:
We need to delete the `cmd/pages` directory also. It contains the pages that the local server displayed.

Reply:
Thanks!